packages/block-editor/src/components/block-binding-support/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * WordPress dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+
+/**
+ * Internal dependencies
+ */
+import withBlockBindingSupport from './with-block-binding-support';
+
+export const BLOCK_BINDINGS_ALLOWED_BLOCKS = {
+	'core/paragraph': [ 'content' ],
+	'core/heading': [ 'content' ],
+	'core/image': [ 'url', 'title', 'alt' ],
+	'core/button': [ 'url', 'text', 'linkTarget' ],
+};
```

Comment:
Is it a good idea to be able to filter this list?

Reply:
No, it’s going to be replaced with a completely different approach. We purposefully want to keep it controlled.

Reply:
Nice to be aware ahead. I hope it will flexible :-).
For instance, a 3PD creates a new block that wants to connect with the source.

Reply:
The challenge is to have a universal way to replace HTML on the server for attributes sourced from HTML. That's the prerequisite for opening it further.